### PR TITLE
Added ORM some tests in controller

### PR DESCRIPTION
**Pull request by marceldobrica**

### CODE REVIEW

**Comment by marceldobrica** [comment]
CodeSniffer: cnp regex expression too long.... but for visibility I have leaved it as it is.

File: src/Controller/ArgumentResolver/UserDtoArgumentValueResolver.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace App\Controller\ArgumentResolver;
+
+use App\Controller\Dto\UserDto;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Controller\ArgumentValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+
+class UserDtoArgumentValueResolver implements ArgumentValueResolverInterface
+{
+    public function supports(Request $request, ArgumentMetadata $argument)
```
Comment (CalinBolea) [inline src/Controller/ArgumentResolver/UserDtoArgumentValueResolver.php]: return types on this class pls

File: src/Controller/UserController.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public function register(UserDto $userDto): Response
+    {
+        $user = User::createFromDto($userDto);
+        $errors = $this->validator->validate($user);
+
+        if (count($errors) > 0) {
+            $errorArray = [];
+            foreach ($errors as $error) {
+                /**
+                 * @var ConstraintViolation $error
```
Comment (CalinBolea) [inline src/Controller/UserController.php]: this kind of annotations (`@var` for an element from a foreach) should go above the foreach and can be wrriten on a single line (i.e. `/** @var Class $object */`)

File: src/Controller/ArgumentResolver/UserDtoArgumentValueResolver.php (hunk 1 above)
Comment (marceldobrica) [reply src/Controller/ArgumentResolver/UserDtoArgumentValueResolver.php]: ok.

File: src/Controller/UserController.php (hunk 2 above)
Comment (marceldobrica) [reply src/Controller/UserController.php]: ok

**Pull request merged**